psi4/src/psi4/libfock/PKmanagers.h

```diff
@@ -153,7 +153,7 @@ class PKManager {
     double cutoff() const { return cutoff_; }
     int nthreads() const { return nthreads_; }
     int nbf() const { return nbf_; }
-    std::shared_ptr<ERISieve> sieve() const { return sieve_; }
+    std::shared_ptr<TwoBodyAOInt> sieve() const { return sieve_; }
```

Comment:
This should be called something like `integral_evaluator`, no?

Reply:
Yeah, that would definitely be sensible, especially because similar objects in other JK classes take names like `eri` or `eri_computer`. I'll be happy to make that change here, too, as a general PKJK cleanup.

Reply:
Done!